Documentation/Index.rst

```diff
@@ -15,43 +8,25 @@ FAL Protect
 .. only:: html
 
 
-    :Classification:
-        fal_protect
-
-    :Version:
-        |release|
-
-    :Language:
-        en
-
-    :Description:
-        Protect everything within :file:`/fileadmin/` based on associated folder and file restrictions (visibility, user
-        groups and dates of publication).
-
-    :Keywords:
-        FAL, files, directories, secure, security, protection
-
-    :Copyright:
-        2020
-
-    :Author:
-        Xavier Perseguers
-
-    :Email:
-        xavier@causal.ch
-
-    :License:
-        This document is published under the Open Publication License
-        available from https://www.opencontent.org/openpub/
-
-    :Rendered:
-        |today|
+   :Classification: fal_protect
+   :Version:        |release|
+   :Language:       en
+   :Description:
+      Protect everything within :file:`/fileadmin/` based on associated folder
+      and file restrictions (visibility, user groups and dates of publication).
+   :Keywords:       FAL, files, directories, secure, security, protection
+   :Copyright:      2020
+   :Author:         Xavier Perseguers
+   :Email:          xavier@causal.ch
+   :License:        This document is published under the Open Publication License
+                    available from https://www.opencontent.org/openpub/
+   :Rendered:       |today|
```

Comment:
I totally dislike this change, I want to keep consistency with all my other manuals

Reply:
:heavy_check_mark: